### DISCUSSION TITLE
Display labels only on hover

### DISCUSSION THREAD

**Discussion by faseidl** [reaviz/reagraph#289] [Q&A]
Hi, does anyone know if there is a way to display labels only on hover? 

(The problem I'm facing is that node labels crowd the graph to the point of distraction. However, it is useful to see the label for specific nodes of interest. Showing the label when hovering over a node would accomplish this.)

Thanks!

**Comment by amcdnl** [comment]
That sounds like a good idea. We try to be smart about labels with counts and zoom but this might be another good option for users. You could probably extend the

```
export type LabelVisibilityType = 'all' | 'auto' | 'none' | 'nodes' | 'edges';
```

to allow this.

**Reply by faseidl** [reply]
Thanks for the suggestion! I'll see what I can figure out.
